packages/@aws-cdk/aws-logs/lib/query-definition.ts

```diff
@@ -0,0 +1,232 @@
+import { Resource } from '@aws-cdk/core';
+import { Construct } from 'constructs';
+import { CfnQueryDefinition } from '.';
+
+
+/**
+ * Properties for a QueryString
+ */
+export interface QueryStringProps {
+  /**
+  * Retrieves the specified fields from log events for display.
+  *
+  * @default no fields in QueryString
```

Comment:
cdk lint doesn't like it, it would like to know the behaviour when it's undefined.

So I have changed it to: `@default undefined no fields in QueryString`